spec/features/new_project_spec.rb

```diff
@@ -59,13 +59,13 @@
     )
   end
 
-  scenario 'records pageviews through Segment.io if ENV variable set' do
+  scenario 'records pageviews through Segment if ENV variable set' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.